invenio_communities/views/communities.py

```diff
@@ -76,15 +80,19 @@ def communities_detail(community=None, logo=None, pid_value=None):
         },
         # Pass permissions so we can disable partially UI components
         # e.g Settings tab
-        permissions=community.has_permissions_to(['update']),
+        permissions=permissions,
         active_menu_tab="search"
     )
 
 @pass_community
 @pass_community_logo
-@require_community_owner
 def communities_settings(community=None, logo=None, pid_value=None):
     """Community settings/profile page."""
+    permissions=community.has_permissions_to(
+        ['update', 'read', 'search_requests', 'search_invites']
+    )
+    if not permissions['can_update']:
```

Comment:
Can you please check if the service is already raising it this? And if not, why?